katello.gemspec

```diff
@@ -46,10 +46,10 @@ Gem::Specification.new do |gem|
   #pulp3
   gem.add_dependency "pulpcore_client", ">= 3.2.1", "< 3.3.0"
   gem.add_dependency "pulp_file_client", ">= 0.2.0", "< 0.3.0"
-  gem.add_dependency "pulp_ansible_client", ">=0.2.0b11", "< 0.2.0b12.dev01584138469"
+  gem.add_dependency "pulp_ansible_client", "> 0.1", "< 0.3.0"
   gem.add_dependency "pulp_container_client", ">= 1.2.0", "< 1.3.0"
   gem.add_dependency "pulp_rpm_client", ">=3.2.0", "< 3.3.0"
-  gem.add_dependency "pulp_2to3_migration_client", ">= 0.2.0b1", "< 0.2.0b2.dev01585075322"
+  gem.add_dependency "pulp_2to3_migration_client", "> 0.2", "< 0.3.0"
```

Comment:
This one needs to be set to `> 0.1` as well.

Reply:
whoops, should be fixed now